wrlx-image/content.md

```diff
@@ -0,0 +1,45 @@
+# What is wrlx-image?
+
+It is short for Wind River Linux image, which is a small docker image built from [Wind River Linux LTS18](https://github.com/WindRiver-Labs/wrlinux-x), it is based on glibc and busybox, can be expanded by package manager opkg. It supports a lot of most commonly used packages, such as mariadb, apache2, gcc and so on. And busybox can be replaced by related tools easily.
+
+%%LOGO%%
+
+# How to use this image
+
+## Usage
+
+Use like you would any other base image:
+
+```dockerfile
+FROM %%IMAGE%%:10.18.0011
```

Comment:
You probably want to consider adding more "rolling" tags to these images, so users can do something like `FROM %%IMAGE%%:10.18` (and thus be able to get updated images by rebuilding instead of having to update their `Dockerfile` every time).